Builds/CMake/RippledCore.cmake

```diff
@@ -668,7 +669,6 @@ target_sources (rippled PRIVATE
   src/ripple/rpc/handlers/ValidatorListSites.cpp
   src/ripple/rpc/handlers/Validators.cpp
   src/ripple/rpc/handlers/WalletPropose.cpp
-  src/ripple/rpc/handlers/Catalogue.cpp
```

Comment:
there was a duplicate, so this is not actually deleting it from the list, but removing the duplicate